README.md

```diff
@@ -1,21 +1,26 @@
 # helm-charts
 
-Helm chart repository containing the jenkins-infra helm charts used in <https://github.com/jenkins-infra/kubernetes-management>
+Helm chart repository containing the jenkins-infra helm charts used [HERE](https://github.com/jenkins-infra/kubernetes-management)
```

Comment:
Using "here" as a link title is a practise that should be avoided. It's an obstacle for accessibility and for non english speakers.

Reply:
Okay I have removed this. Doesn't the README is written in English?

Reply:
Yes it is written in English. But using the "here" title for an hyperlink is not clear for a lot of non english speaker.

What do you think about adding a title instead of `here` such a "the Jenkins infrastructure kubernetes-management repository"?